src/rust/engine/testutil/local_cas/src/main.rs

```diff
@@ -5,34 +5,36 @@ extern crate mock;
 
 use clap::{App, Arg};
 use mock::StubCAS;
+use std::io;
+use std::io::prelude::*;
```

Comment:
It may be nice to ditch the prelude wildcard include, as it makes it slightly harder to see where things come from, but not a big deal either way